prodtests/full-system-test/dpl-workflow.sh

```diff
@@ -421,6 +418,12 @@ if [[ -n ${WORKFLOW_DETECTORS_USE_GLOBAL_READER_TRACKS} ]] || [[ -n ${WORKFLOW_D
   has_detector TOF && has_detector_from_global_reader TOF && add_W o2-tof-reco-workflow "$DISABLE_MC --input-type digits --output-type NONE $HBFINI_OPTIONS"
 fi
 
+if [[ ${ITSMFT_RECO_RERUN_CLUSTERIZER:-0} == 1 ]]; then
```

Comment:
Can I then just change this to `if [[ ${ITSMFT_RECO_RERUN_CLUSTERIZER:-0} == 1 || ${DISABLE_DIGIT_CLUSTER_INPUT:-0} == 1 ]]; then`?

Reply:
It is always a mess....
But if someone sets ITSMFT_RECO_RERUN_CLUSTERIZER=1 + manual settings and then it crashes, I actually would not care. But both should work exclusively.